src/main/java/com/jozufozu/flywheel/backend/mixin/OptionsMixin.java

```diff
@@ -0,0 +1,23 @@
+package com.jozufozu.flywheel.backend.mixin;
+
+import com.jozufozu.flywheel.backend.engine.uniform.Uniforms;
+
+import org.spongepowered.asm.mixin.Mixin;
+import org.spongepowered.asm.mixin.injection.At;
+import org.spongepowered.asm.mixin.injection.Inject;
+import org.spongepowered.asm.mixin.injection.callback.CallbackInfo;
+
+import net.minecraft.client.Options;
+
+@Mixin(Options.class)
+public class OptionsMixin {
+	@Inject(method = "load()V", at = @At("RETURN"))
+	private void onLoad(CallbackInfo ci) {
+		Uniforms.onOptionsUpdate();
+	}
+
+	@Inject(method = "save", at = @At("HEAD"))
+	private void onSave(CallbackInfo ci) {
+		Uniforms.onOptionsUpdate();
+	}
```

Comment:
super nit: these need the `flywheel$` prefix

Reply:
Oops! Fixed now.